package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
remove this file (git rm package.json)

Reply:
also remove package-lock.json (`git rm package-lock.json`)

Reply:
ok looks like i have removed them.